app/models/content_item.rb

```diff
@@ -28,6 +28,8 @@ class ContentItem < ActiveRecord::Base
   NON_RENDERABLE_FORMATS = %w(redirect gone).freeze
   EMPTY_BASE_PATH_FORMATS = %w(contact government).freeze
 
+  has_many :change_notes
```

Comment:
There has been quite a bit of work to avoid content item "knowing" about other objects. see https://github.com/alphagov/publishing-api/blob/master/doc/model.md#general-themes

It does mean that you would have to do a bit of a dance to work this in the same way, so might be something we want to discuss?